endpoints.json

```diff
@@ -25,5 +25,8 @@
         }
       ]
     }
+  },
```

Comment:
I'd like to see some more info on this endpoint - follow the format of the /api/articles example above for this